app/src/main/res/layout/nav_header.xml

```diff
@@ -11,18 +11,19 @@
     <ImageView
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:src="@mipmap/ic_launcher" />
+        android:src="@mipmap/ic_launcher"
```

Comment:
Thanks for replacing hardcoded strings with @Strings , I think there may be a small conflict with my PR as I modified these lines, just a small thing to keep in mind.